src/main/java/gdx/liftoff/ui/dialogs/FullscreenDialog.java

```diff
@@ -23,8 +25,8 @@
  */
 public class FullscreenDialog extends PopTable {
     public static FullscreenDialog fullscreenDialog;
-    private TextButton generateButton;
-    private Table versionTable;
+    private final ArrayList<TextButton> generateButtons = new ArrayList<>();
+    private final ArrayList<Table> versionTables = new ArrayList<>();
```

Comment:
I'm not quite sure what's going on with multiple tables, but I'm guessing it has to do with the fullscreen and standard views. I definitely want updates to be notifying people!